src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -98,5 +100,11 @@ document.addEventListener('DOMContentLoaded', _e => {
   // startLRNHealthCarWidget(commonStore);
   startFeedbackWidget(commonStore);
   // startAnnouncementWidget(commonStore);
-  startVAFooter(commonStore);
+  startVAFooter({
+    handleFooterDidMount: () => {
+      addOverlayTriggers();
+      addFocusBehaviorToCrisisLineModal();
```

Comment:
current modal attaches listeners to buttons in the header and footer to activate the modal; in the spirit of expediency we're reusing this functionality on the new footer component